### DISCUSSION TITLE
Add a new dependency to decoded APK

### DISCUSSION THREAD

**Discussion by radhaVedula** [iBotPeaches/Apktool#2841] [Q&A]
Hi @iBotPeaches ,
I have a small query regarding the usage of apktool. I want to decode an APK , add a new library to the APK, use the classes in new library in the smali files and rebuild it back using apktool. The library I'm using would be in a .jar file. Is it possible to do so using apktool?

Thanks,
Radha Vedula

**Comment by iBotPeaches** [comment]
Yes you could that, but its going to take some work. Nothing in Apktool will help make this easier, but it won't stop you.
